src/components/FeedbackSticky/index.tsx

```diff
@@ -0,0 +1,184 @@
+import { useCallback, useRef, useState } from 'react';
+import {
+  FeedbackContainer,
+  VoteButton,
+  VoteButtonReplace,
+  VoteButtonsContainer,
+  FeedbackText,
+  Divider,
+  ButtonStyles
+} from './styles';
+import { useEffect } from 'react';
+import { trackFeedbackSubmission } from '../../utils/track';
+import { Icon, Button } from '@cloudscape-design/components';
+
+enum FeedbackState {
+  START = 'START',
+  UP = 'UP',
+  DOWN = 'DOWN',
+  HIDDEN = 'HIDDEN'
+}
+
+type Feedback = {
+  vote: boolean;
+  page_path: string;
+  id?: string;
+  comment?: string;
+};
+
+export default function Feedback() {
+  const [state, setState] = useState<FeedbackState>(FeedbackState.START);
+  const feedbackQuestion = 'Did this page help you?';
+  const yesVoteResponse = 'Thanks for the thumbs up!';
+  const noVoteResponse = "We're sorry we let you down";
```

Comment:
@hibler13 - This is the language that was provided in the design - is it the language we'd want live?

Reply:
@katiegoines If the customer doesn't have the option to provide additional feedback details, I'd suggest we make the wording more generic:

feedbackQuestion = 'Was this page helpful?';
yesVoteResponse = 'Thanks for your feedback.'
noVoteResponse = 'Thanks for your feedback.'

Other than the use of "sorry," my main concern with the no vote response is that it begs for an additional free-text feedback option so that the customer can tell us how we fell short of their expectations. Without that, it feels like an empty apology.